### DISCUSSION TITLE
Where I can find parcel v2 transformer for reshadow?

### DISCUSSION THREAD

**Discussion by yurkimus** [parcel-bundler/parcel#5991] [Q&A] [answered]
I want to use [reshadow](https://reshadow.dev/setup) in my project. I tried to search for `@parcel/transformer-reshadow` and doesn't find something.

**Comment by DeMoorJasper** [comment, marked as answer]
There's no plugin for reshadow, but according to their docs they just have a babel and postcss plugin so you could define a `.babelrc` and `.postcssrc` and it should work.

See https://v2.parceljs.org/languages/postcss/ and https://v2.parceljs.org/languages/babel/

**Reply by yurkimus** [reply]
Thanks!
